app/views/admin/rooms/_form.html.haml

```diff
@@ -10,6 +10,6 @@
   .col-md-8
     = semantic_form_for(@room, url: (@room.new_record? ? admin_conference_venue_rooms_path : admin_conference_venue_room_path(@conference.short_title, @room))) do |f|
       = f.input :name, input_html: { autofocus: true}
-      = f.input :size, input_html: {size: 5}
+      = f.input :size,label: 'Capacity', input_html: {size: 5}
```

Comment:
can you please leave a space between _size_ and _label_, so:

`= f.input :size, label: 'Capacity', input_html: {size: 5}`

just for readability :wink: I wonder why our linters are not complaining :thinking: